src/components/BasicLocation.jsx

```diff
@@ -117,10 +134,15 @@ class BasicLocation extends Component {
 
 BasicLocation.defaultProps = {
   classes: null,
+  view: '',
+  props: '',
```

Comment:
Think this might need to be `{}`.

Reply:
Implemented

Reply:
If I make this an object the select does not load correctly as it needs a string to render the initial value.